test/plausible_web/controllers/api/external_controller_test.exs

```diff
@@ -967,7 +967,7 @@ defmodule PlausibleWeb.Api.ExternalControllerTest do
 
       conn
       |> put_req_header("user-agent", @user_agent)
-      |> put_req_header("x-forwarded-for", "982.32.12.1")
+      |> put_req_header("x-forwarded-for", "82.32.12.1")
```

Comment:
not sure if it was a type or intentional, but in case of an invalid ip address, `locus:lookup` would return `{:error, {:invalid_address, "982.32.12.1"}}` which would result in a `raise`

```elixir
** (RuntimeError) failed to lookup ip address "982.32.12.1": {:invalid_address, "982.32.12.1"}
```